### DISCUSSION TITLE
Onedrive installation issue

### DISCUSSION THREAD

**Discussion by khadeeja5858** [abraunegg/onedrive#2614] [Show and tell]
Hi, after downloading the onedrive , this link below does not work for verifying the onedrive. 

Configuring Global Azure AD Endpoints
Authorize this app visiting:

https://login.microsoftonline.com/common/oauth2/v2.0/authorize?client_id=d50ca740-c83f-4d1b-b616-12c519384f0c&scope=Files.ReadWrite%20Files.ReadWrite.all%20Sites.Read.All%20Sites.ReadWrite.All%20offline_access&response_type=code&redirect_uri=https://login.microsoftonline.com/common/oauth2/nativeclient

Enter the response uri:

**Comment by abraunegg** [comment]
@khadeeja5858 
You must open this link in a web browser, then, login and the response URI is what you paste back to the application.

Please read the documentation more carefully as this is explained in detail: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#authorize-the-application-with-your-onedrive-account

**Reply by khadeeja5858** [reply]
resolved. Thankyou

**Reply by joaociocca** [reply]
how did you solved it? when I visit the given link I get to log on to my account and authorize thru MFA, but nothing shows up after it.

keep getting a 0 content-length answer...
![image](https://github.com/abraunegg/onedrive/assets/6473725/efa438aa-c8a1-467c-b349-1bb15fa302cd)

**Reply by abraunegg** [reply]
@joaociocca 
Please re-read the documentation ... you are not following the documentation or the steps required.

**Reply by joaociocca** [reply]
you know what's worse? I've done this before, and I always forget to pay attention to a single line of it, maybe you should give it some highlight for people like me...
"Copy the URI of the blank page into the application."

I ended up setting up rclone instead in the end 🤦‍♂️
